tests/dummy/config/environment.js

```diff
@@ -1,10 +1,9 @@
 /* eslint-env node */
-'use strict';
 
 module.exports = function(environment) {
-  let ENV = {
+  var ENV = {
     modulePrefix: 'dummy',
-    environment,
+    environment: environment,
```

Comment:
It seems strange to change this to ES5, but then also depend on ES6 features (like in tests/helpers/module-for-acceptance.js)